src/helpers/errorhandler.helper.js

```diff
@@ -8,6 +9,11 @@ exports.errorHandler = (err, req, res, _) => {
             error: err.getResponse()
         });
     }
+    if (err instanceof AuthenticationError) {
```

Comment:
This won't be required. When you will extend `CustomError` as `AuthenticationError` this will also get covered in the first if case at line (7/6).

Reply:
I think this would be required as we are adding one extra field in authentication Error.

Reply:
No as you are extending it from `CustomError` it will never reach here it will be handled from the first if block only.

Other then that the extra key `logout:true` we can over ride the `getResponse` method in `AuthenticationError` to get update the value.